pages/concepts/ai-engine/ai-engine-personalities.mdx

```diff
@@ -0,0 +1,73 @@
+import { Callout } from 'nextra/components'
+
+# AI Engine Personalities
+
+## Introduction
+
+This guide explores the **AI Engine personalities** within Fetch.ai's ecosystem. Let's delve into how these personalities enhance user interactions when operating with Ai Engine and DeltaV!
+
+## The AI Engine personalities: Powering Fetch.ai's Intelligence
+
+The [AI Engine ↗️](/concepts/ai-engine/ai-engine-intro) stands as the heart of Fetch.ai's AI infrastructure, translating human language inputs into actionable tasks for a network of [Agents ↗️](/guides/agents/getting-started/whats-an-agent) to fulfill them. The AI Engine bridges the gap between users and AI by leveraging powerful Large Language Models (LLMs) to understand complex requests. Here's how it works:
+
+    1. **User input**: the user provide instructions via [DeltaV ↗️](/concepts/ai-engine/deltav) which facilitates smooth interaction with the AI Engine.
+    2. **Understanding the request**: the AI Engine employs LLMs to analyze the user's request, grasping the intent and identifying the necessary actions.
+    3. **Task delegation**: based on the analysis, the AI Engine breaks down the user's request into executable tasks.
+    4. **Agent coordination**: the AI Engine then orchestrates a network of agents registered on the Fetch Network to fulfill those tasks, ultimately achieving the user's objective.
+
+In order to offer a more engaging and user-friendly experience, the AI Engine is being designed to adopt multiple personality types depending on the user's needs. These personalities essentially act as communication styles, influencing how the AI interacts with you. Imagine having a choice between a friendly and informal assistant or a more professional and to-the-point one. By tailoring the AI's character to users' preferences or application needs, developers can design more natural and effective interactions and create more engaging and effective applications.
```

Comment:
this is not true. Under personality we select different features usually. For example:
Talkative - our v1 version of ai engine
Next Generation - ai engine v2, with a bunch of changes.

So its not about user friendliness and what suits the user. We release new potentialy breaking features into separate personality so people can use it without breaking. Once things cristalize those features go to the default personality.